docs/modules/ROOT/pages/relay.adoc

```diff
@@ -366,9 +394,9 @@ image::defender-relayer-withdraw-funds-screen.png[Defender Relay Withdraw Funds
 [[under-the-hood]]
 == Under the hood
 
-Each Relayer is associated to a private key. When a request to send a transaction is received, the Relayer validates the request, atomically assigns it a nonce, reserves balance for paying for its gas fees, resolves its speed to a gas price, signs it with its private key, and enqueues it for submission to the blockchain. The response is sent back to the client only after this process has finished. Then, the transaction is broadcasted through both Infura and Alchemy for redundancy and retried up to three times in case both APIs are down.
+Each Relayer is associated to a private key. When a request to send a transaction is received, the Relayer validates the request, atomically assigns it a nonce, reserves balance for paying for its gas fees, resolves its speed to a `gasPrice` or `maxFeePerGas`/`maxPriorityFeePerGas` depending on its <<relay-api-reference.adoc#relayer-policies, EIP1559 pricing policy>>, signs it with its private key, and enqueues it for submission to the blockchain. The response is sent back to the client only after this process has finished. Then, the transaction is broadcasted through both Infura and Alchemy for redundancy and retried up to three times in case both APIs are down.
```

Comment:
I feel like these specifics are prone to change (or maybe already have? Id it just Infura/Alchemy?):

"the transaction is broadcasted through both Infura and Alchemy for redundancy and retried up to three times in case both APIs are down."

what do you think about something like... 

```suggestion
Each Relayer is associated to a private key. When a request to send a transaction is received, the Relayer validates the request, atomically assigns it a nonce, reserves balance for paying for its gas fees, resolves its speed to a `gasPrice` or `maxFeePerGas`/`maxPriorityFeePerGas` depending on its <<relay-api-reference.adoc#relayer-policies, EIP1559 pricing policy>>, signs it with its private key, and enqueues it for submission to the blockchain. The response is sent back to the client only after this process has finished. Then, the transaction is broadcasted through multiple node providers for redundancy and retried if necessary.
```

Reply:
Thanks @mok0230, you're right. I tried not to mess with things not strictly related to EIP1559 but this makes sense.
I updated with your comments